common/src/main/java/com/frostwire/search/one337x/One337xSearchPerformer.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Created by Angel Leon (@gubatron), Alden Torres (aldenml)
+ * Copyright (c) 2011-2018, FrostWire(R). All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.frostwire.search.one337x;
+
+import android.util.Log;
+
+import com.frostwire.search.CrawlableSearchResult;
+import com.frostwire.search.SearchMatcher;
+import com.frostwire.search.torrent.TorrentRegexSearchPerformer;
+
+/**
+ * @author gubatron
+ * @author aldenml
+ */
+public final class One337xSearchPerformer extends TorrentRegexSearchPerformer<One337xSearchResult> {
+    public static final String SEARCH_RESULTS_REGEX = "(?is)<a href=\"/torrent/(?<itemId>[0-9]*)/(?<htmlFileName>.*?)\">";
```

Comment:
you could probably grab the link Display Name here:

`(?is)<a href=\"/torrent/(?<itemId>[0-9]*)/(?<htmlFileName>.*?)\">(?<displayName>.*?)</a>`

And pass it to the Temp Result constructor, right now the TempResult constructor is returning null for getDisplayName